package.json

```diff
@@ -24,7 +24,10 @@
     "gatsby-plugin-sitemap": "^2.4.7",
     "gatsby-plugin-styled-components": "3.3.0",
     "gatsby-plugin-typescript": "2.1.7",
+    "gatsby-remark-autolink-headers": "^2.10.0",
+    "gatsby-remark-external-links": "^0.0.4",
     "gatsby-source-filesystem": "2.1.22",
+    "gatsby-transformer-remark": "^2.15.0",
```

Comment:
Since all other deps are fixed versions please fix the version

Reply:
Done